api/projects.ts

```diff
@@ -76,7 +76,9 @@ export async function fetchProject(
   projectName: string
 ): Promise<Project> {
   return http.get(accountId, {
-    url: `${DEVELOPER_PROJECTS_API_PATH}/by-name/${encodeURIComponent(projectName)}`,
+    url: `${DEVELOPER_PROJECTS_API_PATH}/by-name/${encodeURIComponent(
+      projectName
+    )}`,
```

Comment:
Strange, I just pulled `main` and ran `yarn lint` and they seem to be fine on there 🤔

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/188/commits/77b753a1f91c6307b6555f4691d7d1e965e1e36f